docs/index.html

```diff
@@ -0,0 +1,291 @@
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/themes.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/grid.css" />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="cds-theme-zone-white custom-theme">
+
+      <header>
+        <div class="cds--grid layout-lead-space-fixed__container">
+          <img src = "ecosystem-logo.svg" width="350px" alt="Qiskit Ecosystem"/>
+          <p class="title">Explore projects that use or extend Qiskit</p>
+      
+          <cds-button href="https://github.com/qiskit-community/ecosystem">
+            Join the Ecosystem
+          <span slot="icon"
+                style="margin: 2px 0 -2px 0">
+            <svg focusable="false" preserveAspectRatio="xMidYMid meet"
+                  xmlns="http://www.w3.org/2000/svg" fill="currentColor"
+                  width="16" height="16" viewBox="0 0 16 16" aria-hidden="true"
+                  data-v-bdc8f36d="">
+              <path d="M13,14H3c-0.6,0-1-0.4-1-1V3c0-0.6,0.4-1,1-1h5v1H3v10h10V8h1v5C14,13.6,13.6,14,13,14z"></path>
+              <path d="M10 1L10 2 13.3 2 9 6.3 9.7 7 14 2.7 14 6 15 6 15 1z"></path>
+            </svg>
+          </span>
+          </cds-button>
+        </div>
+      </header>
+
+      <div class="content cds--grid layout-lead-space-fixed__container">
+        <h2 class="section">Providers</h2>
+        <p>Run your quantum programs using the following projects</p>
+        <div class="cards cds--row">
+
+          <div class="card cds--col">
+            <cds-tile color-scheme="white">
+              <h2 class="project-title">Project title</h2>
+              <div class="tags">
+                <cds-tag type="purple" title="ibm-quantum" size="sm">Algorithms</cds-tag>
+                <cds-tag type="purple" title="ibm-quantum" size="sm">Physics</cds-tag>
+              </div>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique odio, nostrum voluptatum quidem tempore dolores! Iste vero reprehenderit rerum minima! Laudantium quia nemo aliquid dolorem sit at. Aliquid, consequuntur culpa!</p>
+              <cds-link href="https://ibm.com">
+                Go to repository
+                <span class = "link" slot="icon">
+                  <svg focusable="false" preserveAspectRatio="xMidYMid meet"
+                       xmlns="http://www.w3.org/2000/svg" fill="currentColor"
+                       width="16" height="16" viewBox="0 0 16 16" aria-hidden="true"
+                       data-v-bdc8f36d="">
+                    <path d="M13,14H3c-0.6,0-1-0.4-1-1V3c0-0.6,0.4-1,1-1h5v1H3v10h10V8h1v5C14,13.6,13.6,14,13,14z"></path>
+                    <path d="M10 1L10 2 13.3 2 9 6.3 9.7 7 14 2.7 14 6 15 6 15 1z"></path>
+                  </svg>
+                </span>
```

Comment:
I'd also recommend adding a bit more padding on the left of the icon, atm it looks a bit squished onto the end of the text

Reply:
Done! Now the icons are aligned, and I have added a little bit of margin to the left (4px) like in your example @frankharkins. @javabster do you mean more padding than Frank's screenshot? I can add more if you want :)

In addition, when the screen is small (mobile size) and we have more than one link, they are next to each other instead of one above and the other below (normal size screens). I think with small screens and using the fingers especially, it might be easier to avoid clicking the wrong one with that alignment. Let me know what you think about that as well!